.github/workflows/docker-image-build-push.yml

```diff
@@ -0,0 +1,31 @@
+# Build and push docker image
+name: Docker Image CI
+
+on:
+  push:
+    branches:
+      - production
+      - master
```

Comment:
i led you astray, this needs to be changed to only say `-main`

Reply:
I removed `production` branch and added `main` instead of `master`. Also removed the `staging` tag as I think we'll only need `production` and `latest` tags.